features/grant_document_access.feature

```diff
@@ -0,0 +1,24 @@
+#language: fr
+
+Fonctionnalité: Essayer d'accorder les droits d'édition d'un document à un utilisateur
```

Comment:
Please rephrase the title so that "document" is the last word.
Otherwise the two phrases don't match: "Essayer d'accorder les droits d'édition d'un document à un utilisateur dont on est l'auteur" would mean that you are the author of the user...